openapi.yml

```diff
@@ -932,10 +932,6 @@ paths:
           - IMPORTANT
           - LOW
           - MODERATE
-      - in: query
-        name: flaw__is_major_incident
-        schema:
-          type: boolean
```

Comment:
this breaks API

Reply:
These fields are back, but so far, I have not been able to mark them as deprecated.